ownai/tests/python/AST/parse_cpp_tree_test.py

```diff
@@ -33,11 +31,11 @@ def assert_cpp_files_similar(cpp_file1: str, cpp_file2: str):
         assert token1.kind == token2.kind
         assert token1.spelling == token2.spelling
 
-        try:
-            token1 = next(tokens1)
-            token2 = next(tokens2)
-        except:
-            break
+        try:  #
+            token1 = next(tokens1)  #
+            token2 = next(tokens2)  #
+        except:  #
+            break  #
 
 
 @pytest.mark.parametrize(
```

Comment:
E722 Do not use bare `except`
```suggestion

        try:  #
            token1 = next(tokens1)  #
            token2 = next(tokens2)  #
        except StopIteration:
            break  #


@pytest.mark.parametrize(

```